### DISCUSSION TITLE
add different tags for PCAP import and Monitoring port

### DISCUSSION THREAD

**Discussion by nebra-status** [Security-Onion-Solutions/securityonion#13569] [2.4]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

48

### RAM

256

### Storage for /

300GB

### Storage for /nsm

13TB

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi there,

we're currently running a system that is both monitoring realtime traffic and where pcaps from another network must be imported with so-import-pcap.
As these are 2 different networks, I'd like to add a tag "pcap-import" to the data from the PCAPs during the import.
Is there some smart way to do this with the logstash pipeline?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by Anelka3523** [comment]
I have also same problem. There is a filename123.xml for each filename123.pcap file coming from different networks. I want to add the columns in this xml. thanks

**Comment by TOoSmOotH** [comment]
Do the IP ranges overlap or something?

**Comment by nebra-status** [comment]
Yes, local IP addresses overlap

**Comment by dougburks** [comment]
By default, imported data should already have a few different ways of differentiating it from live data.

```
data_stream.dataset:"import" 
```
![image](https://github.com/user-attachments/assets/540fe6c3-f090-4ace-a435-b30873b55466)

```
event.imported:"true" 
```
![image](https://github.com/user-attachments/assets/7f504c89-3b51-4d6c-8a60-e2eb6607e296)

```
_exists_:import.file
```
![image](https://github.com/user-attachments/assets/e136cfa1-20d7-4899-adb3-eeec3da893af)

```
_exists_:import.id
```
![image](https://github.com/user-attachments/assets/64a3073f-35df-4dfb-8720-b9002e6f00e5)
